H/tests/tests.hs

```diff
@@ -73,5 +73,6 @@ main :: IO ()
 main = do
     -- Normalize internalization settings.
     setEnv "LANG" "C"
+    setEnv "LC_ALL" "C"
```

Comment:
No need to set `LANG` if we're setting `LC_ALL` as it will be set automatically.

Further, maybe we should go with some UTF here? Don't want tests to start chocking when some unicode makes its way in (and I think we should actually have a test with unicode in it).